lib/errors/internaloautherror.js

```diff
@@ -13,6 +13,13 @@ function InternalOAuthError(message, err) {
   Error.call(this);
   Error.captureStackTrace(this, InternalOAuthError);
   this.name = 'InternalOAuthError';
+  if (err && err.data) {
+    try {
+      let errlist = JSON.parse(err.data);
+      message += '\nerror: \n' + errlist['error'] + '\nerror_message: \n' + errlist['error_message']; 
+    } catch(ex) {      
```

Comment:
If you can't JSON parse the error, maybe just log the error data as the message directly so it's not lost.